File: 01 - Sequenciais/sequenciais_3.dart
// 3 - Faça um programa que peça dois números e imprima a soma.
import 'dart:io';

main(){
  stdout.write("Digite um número: ");
  int n1 = int.parse(stdin.readLineSync());
  stdout.write("Digite outro número: ");
  int n2 = int.parse(stdin.readLineSync());

  print("A soma dos números é $n1 e $n2 é ${n1+n2}");
}

File: 01 - Sequenciais/sequenciais_3.js
// 3 - Faça um programa que peça dois números e imprima a soma.

var readline = require('readline');
var num1;
var num2;

var leitor = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

leitor.question("Informe um número? \n", function(answer) {
    var num1 = answer;
    leitor.close();
});
leitor.question("Informe um número? \n", function(answer) {
    var num2 = answer;
    leitor.close();
});

var soma = num1 + num2;
console.log('A soma dos números informados é: ', soma)